Makefile

```diff
@@ -93,24 +88,16 @@ endif
 
 # Enabling debug PRINTF
 ifneq ($(DEBUG),0)
-        DEFINES += HAVE_STACK_OVERFLOW_CHECK
-        SDK_SOURCE_PATH  += lib_stusb lib_stusb_impl
-        DEFINES   += HAVE_IO_USB HAVE_L4_USBLIB IO_USB_MAX_ENDPOINTS=4 IO_HID_EP_LENGTH=64 HAVE_USB_APDU
-
-        ifeq ($(DEBUG),10)
-                $(warning Using semihosted PRINTF. Only run with speculos!)
-                CFLAGS    += -include src/dbg/debug.h
-                DEFINES   += HAVE_PRINTF PRINTF=semihosted_printf
```

Comment:
This is not used?

Reply:
Probably comes from a time where Speculos required this to have PRINTFs working.
It is not required anymore as the regular `mcu_usb_printf` works just fine.